commercial-support.tt

```diff
@@ -0,0 +1,21 @@
+[% WRAPPER layout.tt hideTitle=1 title="Commercial support" menu='nixos' %]
+
+<h1>Commercial support</h1>
+
+<div class="lead">
+  A number of consulting companies are available for professional support.
+</div>
+
+<p>The list is sorted in alphabetical order.</p>
+
+<!-- Please submit a pull request to add your company -->
```

Comment:
Instead of this comment I would prefer a visible edit-link to GitHub. Admittedly this is a nice riddle for the exploration-minded visitor, but unfortunately not everyone bothers to read the source code or to find the repository. Especially in a professional context time is money, so we should make this as easy as we can.

Reply:
good idea, but that is already an improvement. it's small, so i will add it, but i want get this finished and come to it later again